web/src/views/DandisetLandingView/DownloadDialog.vue

```diff
@@ -125,14 +125,14 @@ function formatDownloadCommand(identifier: string, version: string): string {
     const baseUrl = import.meta.env.VITE_APP_DANDI_API_ROOT === 'https://staging-api.lincbrain.org/api/'
       ? 'https://staging.lincbrain.org/dandiset/'
       : 'https://lincbrain.org/dandiset/';
-    return `lincbrain download ${baseUrl}${identifier}/draft`;
+    return `dandi download ${baseUrl}${identifier}/draft`;
   }
 
   if (!version) {
-    return `lincbrain download DANDI:${identifier}`;
+    return `dandi download DANDI:${identifier}`;
```

Comment:
It looks like this if-clause returns the latest published version.  Since the `DANDI:` [resource identifier](https://dandi.readthedocs.io/en/latest/ref/urls.html) is only available for the main archive instance, we will need to replace the command.  I am not sure which resource identifier to use to return the latest published version.

Reply:
What might you mean here exactly? Let me know if the code I wrote is appropriate for what you had in mind: [updates per Kabi's comments](https://github.com/lincbrain/linc-archive/pull/276/commits/21bf2a6f1cd3d2ef76f8b1bc9161f66b93ac78c1)

Reply:
Thanks for the discussion.  We decided to remove the radio button for now, pending further discussion with the DANDI team.